Source/WebKit/NetworkProcess/storage/BackgroundFetchStoreImpl.h

```diff
@@ -42,13 +42,15 @@ class SWServer;
 namespace WebKit {
 
 class NetworkStorageManager;
+struct BackgroundFetchState;
 
 class BackgroundFetchStoreImpl :  public WebCore::BackgroundFetchStore {
 public:
     static Ref<BackgroundFetchStoreImpl> create(WeakPtr<NetworkStorageManager>&& manager, WeakPtr<WebCore::SWServer>&& server) { return adoptRef(*new BackgroundFetchStoreImpl(WTFMove(manager), WTFMove(server))); }
     ~BackgroundFetchStoreImpl();
 
     void getAllBackgroundFetchIdentifiers(CompletionHandler<void(Vector<String>&&)>&&);
+    void getBackgroundFetchState(const String&, CompletionHandler<void(std::optional<BackgroundFetchState>&&)>&&);
```

Comment:
Why are we using get prefixes for getters?

Reply:
It is an async getter, hence why the get, although other prefixes like retrieve are used as well.